tests/contrib/fastapi/test_fastapi_patch.py

```diff
@@ -13,10 +16,22 @@ def assert_module_patched(self, fastapi):
         self.assert_wrapped(fastapi.applications.FastAPI.__init__)
         self.assert_wrapped(fastapi.routing.serialize_response)
 
+        self.assert_wrapped(starlette.applications.Starlette.__init__)
```

Comment:
👍🏻 or I wonder... maybe @Kyle-Verhoog knows... can we subclass the `TestStarlettePatch` test here?

Then we could do:

```python
    def assert_not_module_patched(self, fastapi):
        super().assert_not_module_patched(starlette)
        self.assert_not_wrapped(fastapi.applications.FastAPI.__init__)
        self.assert_not_wrapped(fastapi.routing.serialize_response)      
```

... "ish" ?